.github/workflows/ci.yml

```diff
@@ -0,0 +1,63 @@
+name: CI
+
+on: [push]
```

Comment:
Add `pr` to it? When people with forks make a PR, it won't trigger the `push` job since it's not a push on this repo

Reply:
Are you sure about that? We use this pattern with the other repos, kotlinpoet uses this too

Reply:
I think so. I saw this in another repo. We can merge as is and i can test in a followup.